mne/preprocessing/tests/test_eeglab_infomax.py

```diff
@@ -61,14 +61,25 @@ def test_mne_python_vs_eeglab():
     """
     random_state = 42
 
-    list_ch_types = ['eeg', 'mag']
+    methods = ['infomax', 'infomax', 'extended_infomax', 'extended_infomax']
+    list_ch_types = ['eeg', 'mag', 'eeg', 'mag']
 
-    for ch_type in list_ch_types:
+    for method, ch_type in zip(methods, list_ch_types):
 
-        if ch_type == 'eeg':
-            eeglab_results_file = 'eeglab_infomax_results_eeg_data.mat'
-        elif ch_type == 'mag':
-            eeglab_results_file = 'eeglab_infomax_results_meg_data.mat'
+        if method == 'infomax':
+            if ch_type == 'eeg':
+                eeglab_results_file = 'eeglab_infomax_results_eeg_data.mat'
+            elif ch_type == 'mag':
+                eeglab_results_file = 'eeglab_infomax_results_meg_data.mat'
+
+        elif method == 'extended_infomax':
+
+            if ch_type == 'eeg':
+                eeglab_results_file = \
+                    'eeglab_extended_infomax_results_eeg_data.mat'
+            elif ch_type == 'mag':
+                eeglab_results_file = \
+                    'eeglab_extended_infomax_results_meg_data.mat'
```

Comment:
more pythonic not to use a line break but parenthesis:
"""Python
eeglab_results_file =  ('eeglab_extended_infomax_results_meg_'
                                    'data.mat')
"""